tools/cldr-code/src/main/resources/org/unicode/cldr/util/data/dl_iso_table_a3.xml

```diff
@@ -1,1136 +1,1143 @@
+			<CtryNm>BELARUS</CtryNm>
+			<CcyNm>Russian Ruble</CcyNm>
+			<Ccy>RUR</Ccy>
+			<CcyNbr>810</CcyNbr>
+			<WthdrwlDt>1994-06</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>BELGIUM</CtryNm>
+			<CcyNm>Convertible Franc</CcyNm>
+			<Ccy>BEC</Ccy>
+			<CcyNbr>993</CcyNbr>
+			<WthdrwlDt>1990-03</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>BELGIUM</CtryNm>
+			<CcyNm>Belgian Franc</CcyNm>
+			<Ccy>BEF</Ccy>
+			<CcyNbr>056</CcyNbr>
+			<WthdrwlDt>2002-03</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>BELGIUM</CtryNm>
+			<CcyNm>Financial Franc</CcyNm>
+			<Ccy>BEL</Ccy>
+			<CcyNbr>992</CcyNbr>
+			<WthdrwlDt>1990-03</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>BOLIVIA</CtryNm>
+			<CcyNm>Peso boliviano</CcyNm>
+			<Ccy>BOP</Ccy>
+			<CcyNbr>068</CcyNbr>
+			<WthdrwlDt>1987-02</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>BOSNIA AND HERZEGOVINA</CtryNm>
+			<CcyNm>Dinar</CcyNm>
+			<Ccy>BAD</Ccy>
+			<CcyNbr>070</CcyNbr>
+			<WthdrwlDt>1998-07</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>BRAZIL</CtryNm>
+			<CcyNm>Cruzeiro</CcyNm>
+			<Ccy>BRB</Ccy>
+			<CcyNbr>076</CcyNbr>
+			<WthdrwlDt>1986-03</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>BRAZIL</CtryNm>
+			<CcyNm>Cruzado</CcyNm>
+			<Ccy>BRC</Ccy>
+			<CcyNbr>076</CcyNbr>
+			<WthdrwlDt>1989-02</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>BRAZIL</CtryNm>
+			<CcyNm>Cruzeiro</CcyNm>
+			<Ccy>BRE</Ccy>
+			<CcyNbr>076</CcyNbr>
+			<WthdrwlDt>1993-03</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>BRAZIL</CtryNm>
+			<CcyNm>New Cruzado</CcyNm>
+			<Ccy>BRN</Ccy>
+			<CcyNbr>076</CcyNbr>
+			<WthdrwlDt>1990-03</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>BRAZIL</CtryNm>
+			<CcyNm>Cruzeiro Real</CcyNm>
+			<Ccy>BRR</Ccy>
+			<CcyNbr>987</CcyNbr>
+			<WthdrwlDt>1994-07</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>BULGARIA</CtryNm>
+			<CcyNm>Lev A/52</CcyNm>
+			<Ccy>BGJ</Ccy>
+			<CcyNbr>100</CcyNbr>
+			<WthdrwlDt>1989 to 1990</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>BULGARIA</CtryNm>
+			<CcyNm>Lev A/62</CcyNm>
+			<Ccy>BGK</Ccy>
+			<CcyNbr>100</CcyNbr>
+			<WthdrwlDt>1989 to 1990</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>BULGARIA</CtryNm>
+			<CcyNm>Lev</CcyNm>
+			<Ccy>BGL</Ccy>
+			<CcyNbr>100</CcyNbr>
+			<WthdrwlDt>2003-11</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>BURMA </CtryNm>
+			<CcyNm>Kyat</CcyNm>
+			<Ccy>BUK</Ccy>
+			<CcyNbr>104</CcyNbr>
+			<WthdrwlDt>1990-02</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>CROATIA</CtryNm>
+			<CcyNm>Croatian Dinar</CcyNm>
+			<Ccy>HRD</Ccy>
+			<CcyNbr>191</CcyNbr>
+			<WthdrwlDt>1995-01</WthdrwlDt>
+		</HstrcCcyNtry>
+		<HstrcCcyNtry>
+			<CtryNm>CROATIA</CtryNm>
+			<CcyNm>Croatian Kuna</CcyNm>
+			<Ccy>HRK</Ccy>
+			<CcyNbr>191</CcyNbr>
+			<WthdrwlDt>2015-06</WthdrwlDt>
+		</HstrcCcyNtry>

```

Comment:
Had to "hide whitespace changes" to review this effectively.

Reply:
That's odd. I don't see extraneous whitespace.